src/Regulator.h

```diff
@@ -51,12 +51,26 @@
 using std::vector;
```

Comment:
Not changed currently but could you remove this. It's not a good idea in a header file because it can change the behavior of external code.